out-adapter-persistence-jpa/src/test/java/gohigher/application/entity/ApplicationRepositoryTest.java

```diff
@@ -0,0 +1,137 @@
+package gohigher.application.entity;
+
+import static gohigher.fixture.ApplicationFixture.*;
+import static gohigher.fixture.ProcessFixture.*;
+import static org.assertj.core.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import jakarta.persistence.EntityManager;
+
+@DisplayName("ApplicationRepository 클래스의")
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+class ApplicationRepositoryTest {
+
+	@Autowired
+	ApplicationRepository applicationRepository;
```

Comment:
```suggestion
	private ApplicationRepository applicationRepository;
```
테스트라서 큰 의미는 없지만... 다른 코드와 형식을 맞춰보면 어떨까요!

Reply:
테스트여도 중요하죠..!!
해당 클래스에도 접근제어자를 추가했습니다!